src/api/models.py

```diff
@@ -22,81 +22,135 @@
+                     alias="schema__url")
 
-    source: str = Field(description="Name of the source this signal belongs to.")
+    source: str = Field( 
+        description="Name of the source this signal belongs to.",
+        alias="dct__source")
 
     quality: Quality = Field(
         sa_column=Column(
             Enum(Quality, values_callable=lambda obj: [e.value for e in obj])
         ),
-        description="Quality flag for this signal.",
+        alias="dqv__QualityAnnotation",
+        description="Quality flag for this signal."
     )
 
     shape: Optional[List[int]] = Field(
         sa_column=Column(ARRAY(Integer)),
-        description="Shape of each dimension of this signal. e.g. [10, 100, 3]",
+        description="Shape of each dimension of this signal. e.g. [10, 100, 3]"
     )
 
     provenance: Optional[Dict] = Field(
         default={},
         sa_column=Column(JSONB),
-        description="Information about the provenance graph that generated this signal in the PROV standard.",
+        description="Information about the provenance graph that generated this signal in the PROV standard."
     )
 
     units: Optional[str] = Field(
+      
         description="The units of data contained within this dataset."
     )
 
     description: str = Field(
+       
         sa_column=Column(Text), description="The description of the dataset."
     )
 
     signal_type: SignalType = Field(
         sa_column=Column(
             Enum(SignalType, values_callable=lambda obj: [e.value for e in obj])
         ),
-        description="The type of the signal dataset. e.g. 'Raw', 'Analysed'",
+        description="The type of the signal dataset. e.g. 'Raw', 'Analysed'"
     )
 
     dimensions: Optional[List[str]] = Field(
         sa_column=Column(ARRAY(Text)),
-        description="The dimension names of the dataset, in order. e.g. ['time', 'radius']",
+        description="The dimension names of the dataset, in order. e.g. ['time', 'radius']"
     )
 
     shot: "ShotModel" = Relationship(back_populates="signals")
 
+    rights: str = Field(
+        alias="dct__rights",
+        description="The right of usage and ownership of this dataset.",
+        sa_column_kwargs = {"server_default": "The right of using this datasets"},
+    )
+    language: str = Field(
+        alias="dct__language",
+        description="The the language of this dataset.",
+        sa_column_kwargs = {"server_default": "en"}
+    )
+
+    format: str = Field(
+        alias="dct__format",
+        description="The format of data contained within this dataset.",
+        sa_column_kwargs = {"server_default": "the format of this dataset"},
+    )
+    location: str = Field(
+        alias="dct__location", 
+        description="The spatial location of the device that generated this dataset.",
+        sa_column_kwargs = {"server_default": "the spatial location of the device that generates this datasets"},
+    )
```

Comment:
Shouldn't the default be a value, rather than a description?

Same question for other entries added?